microsoft/atlas-design
# convert spacing atomics to use logical properties

Task: task-533674

Link: preview-315

Time to finally do it! After this PR spacing properties will continue to use their traditional (classic?) names, but  under the hood will use css logical properties. This is one of several major steps away from the previous way of solving the multi-reading-direction problem (interpolation like `#{$user-left}`).

## Testing

1. Checkout the spacing documentation for `main`: https://design.docs.microsoft.com/atomics/spacing.html
1. Checkout the spacing documentation for this PR: https://design.docs.microsoft.com/pulls/315/atomics/spacing.html

They should look the same.

## Additional information

![image](https://user-images.githubusercontent.com/30843002/148618838-f0e6cec4-4dbe-4213-b597-ab645affda17.png)
